mw-webapp/src/component/button/Button.module.scss

```diff
@@ -1,14 +1,21 @@
 @import "src/styles/variables";
 
 .button {
-  width: 100px;
-  height: 40px;
+  padding: 13px 17px;
+  border: none;
+  border-radius: $primaryBorderRadius;
+  background-color: $inputInitial;
+  color: $whiteText;
+  font-family: $primaryFontFamily;
+  font-size: $primaryFontSize;
+  transition-duration: $primaryTransitionDuration;
 
   &:hover {
+    background-color: $hover;
```

Comment:
llet's for color use naming: $primary, secondary, etc

Reply:
Right now we don't know what colors we can name primary or secondary because we don't have a whole design. I proceeded from the premise that we have only styles for some different types of input (checkbox, textarea, text) and they may have different states same as button.

Reply:
Done